src/vs/workbench/contrib/terminalContrib/chatAgentTools/browser/runInTerminalTool.ts

```diff
@@ -315,18 +356,22 @@ export class RunInTerminalTool extends Disposable implements IToolImpl {
 				}
 				throw e;
 			} finally {
+				this._logService.trace('runInTerminalTool#outputLineCount, ', outputAndIdle?.output ? count(outputAndIdle.output, '\n') : 0);
+				this._logService.trace('runInTerminalTool#pollDurationMs, ', outputAndIdle?.pollDurationMs);
```

Comment:
A debug log as close as possible in format to foreground terminals would be best:

```
this._logService.debug(`RunInTerminalTool: Finished \`${strategy.type}\` execute strategy with exitCode \`${executeResult.exitCode}\`, result.length \`${executeResult.result.length}\`, error \`${executeResult.error}\``);
```